### ISSUE TITLE
Allow quick switching between a set of maps

### ISSUE THREAD

**Issue by sas8jedi** [coldino/Larkator#36]
It would be of great convenience to be able switching between different maps on a cluster without going to the settings and manually re-choosing the map paths. It would be especially useful while looking for dinos on several maps at the same time.

**Comment by sixokig** [comment]
I agree, great convenience when you play through multiple maps to transfer dinos.
